pkg/controller.v1alpha3/suggestion/composer/composer.go

```diff
@@ -21,7 +21,8 @@ import (
 )
 
 const (
-	defaultInitialDelaySeconds = 10
+	defaultInitialDelaySeconds = 60
```

Comment:
do we need timeout changes again? or are they harmless?

Reply:
It does not have any side effect. I think.

Reply:
It may make suggestion not available until 60sec, should we revert to 10?